ov/server_hardware_type.go

```diff
@@ -70,6 +70,41 @@ type BiosSetting struct {
 	WarningText     string       `json:"warningText,omitempty"`     // "warningText":""
 }
 
+type serverDisplaySchema struct {
```

Comment:
Any reason you have kept the struct name in smallcase. Smallcase wont allow it to be exported.

Reply:
done